backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -11,18 +16,161 @@ export class AdminResolver {
   async searchUsers(@Arg('searchText') searchText: string): Promise<UserAdmin[]> {
     const loginUserRepository = getCustomRepository(LoginUserRepository)
     const loginUsers = await loginUserRepository.findBySearchCriteria(searchText)
-    const users = loginUsers.map((loginUser) => {
-      const user = new UserAdmin()
-      user.firstName = loginUser.firstName
-      user.lastName = loginUser.lastName
-      user.email = loginUser.email
-      user.creation = [
-        (Math.floor(Math.random() * 50) + 1) * 20,
-        (Math.floor(Math.random() * 50) + 1) * 20,
-        (Math.floor(Math.random() * 50) + 1) * 20,
-      ]
-      return user
-    })
+    const users = await Promise.all(
+      loginUsers.map(async (loginUser) => {
+        const user = new UserAdmin()
+        user.firstName = loginUser.firstName
+        user.lastName = loginUser.lastName
+        user.email = loginUser.email
+        user.creation = await getUserCreations(loginUser.id)
```

Comment:
this looks scarily slow due to many many database request - for now ok but will require a refactor later

Reply:
We need pagination for this!

Reply:
https://github.com/gradido/gradido/pull/1135#discussion_r758425215

Reply:
Will be done in a separat branch